apps/api/v2/src/modules/oauth-clients/oauth-client.repository.ts

```diff
@@ -59,7 +59,7 @@ export class OAuthClientRepository {
   }
 
   async getOAuthClientWithRefreshSecret(clientId: string, clientSecret: string, refreshToken: string) {
-    return await this.dbRead.prisma.platformOAuthClient.findFirst({
+    return this.dbRead.prisma.platformOAuthClient.findFirst({
```

Comment:
Why no await here?

Reply:
It's already a Promise, it's being awaited in the actual usage.